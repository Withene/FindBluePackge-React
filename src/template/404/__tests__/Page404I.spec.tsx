import { render, screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import { Page404 } from '../index';
import { BrowserRouter } from 'react-router-dom';
import '@testing-library/jest-dom';

describe('FindYouPackger', () => {
  it('Should able to Render Find', async () => {
    render(
      <BrowserRouter>
        <Page404 />
      </BrowserRouter>,
    );
    expect(screen.getByAltText('Logo FindBluePackge')).toBeInTheDocument();
    expect(screen.getByAltText('PageNoutFound')).toHaveAttribute('src', '404Page.svg');
    expect(screen.getByAltText('PageNoutFound')).toBeInTheDocument();
  });
  it('should match snapshot', () => {
    const { container } = render(
      <BrowserRouter>
        <Page404 />
      </BrowserRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
