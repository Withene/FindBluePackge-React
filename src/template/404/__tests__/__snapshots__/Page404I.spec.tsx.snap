// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FindYouPackger should match snapshot 1`] = `
<div>
  <div
    class="sc-bdfBQB dyHRYR"
  >
    <header
      class="sc-gsTEea fgNrYl"
    >
      <img
        alt="Logo FindBluePackge"
        src="BlueState.svg"
      />
    </header>
    <div
      class="sc-eCstlR hqxUkG"
    >
      <div
        class="sc-jSgvzq lJyuW"
      >
         
        <img
          alt="PageNoutFound"
          src="404Page.svg"
        />
        <h1
          class="sc-dlfnuX bsNShh"
        >
          Ops.. Oque Você está fazendo aqui ?
        </h1>
        <a
          class="sc-hKgJUU gjcdeK"
          href="/"
        >
          Retornar a Home
        </a>
      </div>
    </div>
  </div>
  <div
    class="sc-gKseQn"
  >
    <div
      class="sc-iBPTik jivZgG"
      height="15"
    >
      <img
        alt="Imagem do Footer"
        class="sc-fubCzh iTtKTe"
        src="test.svg"
      />
    </div>
  </div>
</div>
`;
